environment/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "aind-vr-foraging-primary-data-nwb-packaging"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "aind-behavior-vr-foraging[data]",
```

Comment:
This is on me, but for now please lock this to a specific commit. I will make a release soon and you can lock to it then.